mods/mc_worldmanager/commands.lua

```diff
@@ -245,6 +250,72 @@ commands["define"] = {
         Realm:Restore(newRealm)
     end }
 
+commands["privs"] = {
+    func = function(name, params)
+        local operation = tostring(params[1])
+        local realmID = tonumber(params[2])
+        local privilege = tostring(params[3])
+
+        if (operation == nil or operation == "") then
```

Comment:
`tostring(nil)` on lines [255](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/100/files#diff-438bd03cc76fd81ad9448a1f1d70fcb5c719286e2841b32e95fb546e6d172d73R255) and [257](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/100/files#diff-438bd03cc76fd81ad9448a1f1d70fcb5c719286e2841b32e95fb546e6d172d73R257) produces `"nil"`, which means that the comparisons on lines [259](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/100/files#diff-438bd03cc76fd81ad9448a1f1d70fcb5c719286e2841b32e95fb546e6d172d73R259) and [289](https://github.com/ubc-minetest-classroom/Minetest_Classroom/pull/100/files#diff-438bd03cc76fd81ad9448a1f1d70fcb5c719286e2841b32e95fb546e6d172d73R289) (`"nil" == nil`) will return `false` if `params[1]` or `params[3]` is `nil`

Two patch suggestions are provided below - alternatively, a nil check could be added to lines 255 and 257

```suggestion
        if (operation == "nil" or operation == "") then
```

```suggestion
        if (params[1] == nil or operation == "") then
```